test/integration/guest_user_test.rb

```diff
@@ -67,24 +67,54 @@ def setup
     refute page.has_content?("Create Category")
   end
 
-  test "an unauthorised user can view a single item's page" do
-    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
+  # test "an unauthorised user can view a single items page" do
+  # item = Item.create(title: 'coffee', description: 'black nectar of the
+  # gods', price: 1200)
+  #
+  #   visit root_url
+  #   click_link_or_button('Menu')
+  #   click_link_or_button('coffee')
+  #
+  #   within('#title') do
+  #     assert page.has_content?('coffee')
+  #   end
+  #
+  #   within('#description') do
+  #     assert page.has_content?('black nectar of the gods')
+  #   end
+  #
+  #   within('#price') do
+  #     assert page.has_content?('$12.00')
+  #   end
+  # end
+
+  test "an unauthorised user can view a tenants page which only shows their
+  products" do
+    item = Item.create(title: 'coffee',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.